src/Chocolatey.PowerShell/Helpers/EnvironmentHelper.cs

```diff
@@ -166,32 +170,38 @@ public static void SetVariable(PSCmdlet cmdlet, string name, EnvironmentVariable
 
                 cmdlet.WriteDebug($"Registry type for {name} is/will be {registryType}");
 
-                if (string.IsNullOrEmpty(value))
-                {
-                    registryKey.DeleteValue(name, throwOnMissingValue: false);
-                }
-                else
+                if (cmdlet.ShouldProcess(name, $"set {scope} environment variable"))

```

Comment:
Could we be consistent with these `ShouldProcess` calls on the capitalization used?

See previous comment for suggested style of first letter capitalized. (Does not apply to `name` in this context)

Reply:
Sorted!